code/src/WijmoProvider/Features/ColumnFilter.ts

```diff
@@ -67,11 +68,14 @@ namespace WijmoProvider.Feature {
                 ).length > 0
             );
         }
+
         public activate(columID: string): void {
             const column = GridAPI.ColumnManager.GetColumnById(columID);
 
-            this._filter.getColumnFilter(column.provider).filterType =
-                wijmo.grid.filter.FilterType.Both;
+            this._filter.getColumnFilter(column.provider).filterType = this
+                ._grid.config.serverSidePagination
+                ? wijmo.grid.filter.FilterType.Condition
+                : wijmo.grid.filter.FilterType.Both;
```

Comment:
It looks good! 🔝
If we end up using this condition multiple times, please add it into a new getter or a method that will return FilterType.Condition or FilterType.Both depending on the serverSidePagination or any other condition that we might end up adding into the formula. (I'm seeing this again inside the setState method)